support-workers/src/main/scala/com/gu/stripe/StripeBrand.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.stripe
+
+import io.circe.Decoder
+
+// https://stripe.com/docs/api/payment_methods/object#payment_method_object-card-brand
+// See CreditCardType at https://www.zuora.com/developer/api-reference/#operation/Object_POSTPaymentMethod
+sealed abstract class StripeBrand(val value: String, val zuoraCreditCardType: Option[String])
```

Comment:
I'm not sure why i put zuora stuff in the stripe card type, pattern match in the zuora code would have worked